### ISSUE TITLE
Unit Test (E2E) Tests Fail.

### ISSUE THREAD

**Issue by crazeydave** [MetalMynds/Ghostice#1]
If you have a problem running the unit tests this could be due to two embedded example binaries used in the E2E testing.

Their source code is not part of the project yet, however it is available on the internet and will be part of the project soon.

If your unsure and want to check please look at the WipFlash GUI Automation project https://github.com/google-code-export/wipflash from which the WPF one is compiled. 
I created the second to get a differential view of speed WPF vs WinForms it's code is part of a sister project called FlatGlass. https://github.com/MetalMynds

These binaries can become 'blocked' if copied onto a machine from the internet or network share. 
They can be safely unblocked.

They live in the unit test project (see project suffix) in the example folder.
Every file (including .exe .dll and .xml files) should be unblocked, by opening the windows explorer properties window (select file and press ALT + ENTER) and clicking the 'unblock' button.
